<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for http-exceptions</title>
  </head>
  <body>
    <h1>Links for http-exceptions</h1>
<a href="https://files.pythonhosted.org/packages/dd/9f/5a5e5925859655b9a8b1910f24e1d13bdb21ecd5c067d03b875882c80c52/http_exceptions-0.0.1-py3-none-any.whl#sha256=1b7d1f5140b33ac243b2aff38e060ebf9063509346073acc9a78b178414459ce" data-requires-python="~=3.7" >http_exceptions-0.0.1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/8d/87/822e5bc038e89803f28d4f113a27b89ce1d6a11e1cc542ebbd0573a58ea4/http_exceptions-0.0.3-py3-none-any.whl#sha256=7251e225eabcae708d10b73061d09b72ba33dd910062c9ba0beaef31924b7a08" data-requires-python="~=3.7" >http_exceptions-0.0.3-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/ed/7c/06ffb6ef3a316f6d7881dfe2d56e4159e144e0b2535690711853067180ab/http_exceptions-0.0.4-py3-none-any.whl#sha256=ecc5a189f2f7bc50c0071113f27b3ac171f3528c695a9cedcac4972323413aff" data-requires-python="~=3.7" >http_exceptions-0.0.4-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/aa/35/2200b4e2d332f868ad90d6fd7916e783e895d913b38f59479b6b7ffe4cdc/http_exceptions-0.0.5-py3-none-any.whl#sha256=d224be79ebac2d7b7cf7d908da5493349bacf4250eef0fa8e136a61d16442c75" data-requires-python="~=3.7" >http_exceptions-0.0.5-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/aa/3d/fee793e31b63a01df9f54c84c76b60f0cf8e197a791345e5bfbf9aec7568/http_exceptions-0.0.6-py3-none-any.whl#sha256=de53e8142a9bf2fe0a6df73ae43c706d6f0d180030c149941d5a29134c163233" data-requires-python="&gt;=3.7" >http_exceptions-0.0.6-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/2f/ce/575281779f7bec882ff5c57beb487e0d01165468bcb30b07143478ee4f66/http_exceptions-0.1.0-py3-none-any.whl#sha256=e5ffb2a4d32c4a154c36e4c161879d34b394122049e4cd9c22a410a5a99ec20b" data-requires-python="&gt;=3.7" >http_exceptions-0.1.0-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/6f/d7/8bdbdcf9d9659bf0ca40a309d559c4fbdd47adf85be973d3c7aa0a91a91c/http_exceptions-0.1.1-py3-none-any.whl#sha256=5603f15d625b3b70da087a37b1a71329593e172de51a1ce72bfb415497551704" data-requires-python="&gt;=3.7" >http_exceptions-0.1.1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/5b/f6/a148fad189467bf6e7b985129e7e82266d35ca3314c27296fbbc62864df1/http_exceptions-0.1.2-py3-none-any.whl#sha256=081d7016418fc01c965f1c98aec7bd9357bb929c7ec78772e8f74b0cf94d6a90" data-requires-python="&gt;=3.7" >http_exceptions-0.1.2-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/aa/ab/645f9dd2fc113e48d3235e4f97fe4f7a96b86622a1682d396cb7ea8a09ae/http_exceptions-0.2.0-py3-none-any.whl#sha256=9ac768d707caabdf882f9eabf40f34d900f88cc399a64ef6db22015fe9553f50" data-requires-python="&gt;=3.7" >http_exceptions-0.2.0-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/8f/a4/b0f9174a156a77662e91c8cf85455844ae5b2a9033b00794d78f0325875f/http_exceptions-0.2.1-py3-none-any.whl#sha256=d2140d8d86fe4bb8823b1a41bcf00405323b2dfa9c5188b2fc2b6a262d71cffa" data-requires-python="&gt;=3.7" >http_exceptions-0.2.1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/ee/c6/368fee213e44ccdac2e9089e0f22393dc54df8d3dacb99cf6ea00ddbd616/http_exceptions-0.2.2-py3-none-any.whl#sha256=fcff0fc728a8238c3b78975f2b13f7a3d1e43ac47a106c4ee180f15e16e7696e" data-requires-python="&gt;=3.7" >http_exceptions-0.2.2-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/16/ea/8484ad35bfcfcf66a5d87051103f0ad17cbbfddd9b479170571106e2d179/http_exceptions-0.2.3-py3-none-any.whl#sha256=826d17cc330318bb95fde14ce9031af4a0b4e305fa3070013785ce942550512a" data-requires-python="&gt;=3.7" >http_exceptions-0.2.3-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/55/0d/d6de973cfb26bd5de8e31fb51ab43c13f24bad38bbf588806fd226652eb4/http-exceptions-0.2.4.tar.gz#sha256=59ca0fd3c7f20adaba4d2e9e5e6da037191c26713c2219f006d1a433f6b3f7e1" data-requires-python="&gt;=3.7,&lt;4.0" >http-exceptions-0.2.4.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/9a/7f/7c948ea948484090ce27bc9ddd05064fef8350f61e8ac8155127746ad9f1/http_exceptions-0.2.4-py3-none-any.whl#sha256=08478b0d981fb685edd7532332e669afd4c19ab0c4057cdc06957fa50e06d64b" data-requires-python="&gt;=3.7,&lt;4.0" >http_exceptions-0.2.4-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/ad/37/cea71cd1f18142a0fc017155930f27485de49e6495c1dd2805dfc603db17/http-exceptions-0.2.5.tar.gz#sha256=9b30f16383e526202cd6bab263f0a157ab86017ae5fdabe98e8f165f7fa2472a" data-requires-python="&gt;=3.7,&lt;4.0" >http-exceptions-0.2.5.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/9a/ff/ab850b4c4c9d1b6c72eb840c89bef2855e7b344854d968d10fa981f70969/http_exceptions-0.2.5-py3-none-any.whl#sha256=2f67222fdec14ba3d1a9ab9eac04770d29e498b21ef88c162318cb71cffb9b27" data-requires-python="&gt;=3.7,&lt;4.0" >http_exceptions-0.2.5-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/53/cf/03cfe8a0feddce81e6389b7ec5123c48bf47cd749986f1989fda4e00e8cb/http-exceptions-0.2.6.tar.gz#sha256=545313a497dcdd16b8412ca6b2d30ab3a354ede321f80f5e17e7bb12a0828ff9" data-requires-python="&gt;=3.7,&lt;4.0" >http-exceptions-0.2.6.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/49/b0/ec8a3aa7b6e862f82387873c5df303d5e62722d7044bcdebab71635b24ba/http_exceptions-0.2.6-py3-none-any.whl#sha256=577ff5a947c33be8c6a9ea466b20dd13d9bbb8eff03d42716a9113d21a474fb9" data-requires-python="&gt;=3.7,&lt;4.0" >http_exceptions-0.2.6-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/92/39/58b89c7aba0c180b0f5cbb7037303a16ecc128d502a46bfe1c6f9b5610bd/http-exceptions-0.2.9.tar.gz#sha256=0b3ed1937cbf08108d6a7f3dd4db13ad247325bed1843850422300235458fa5b" data-requires-python="&gt;=3.7,&lt;4.0" >http-exceptions-0.2.9.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/fb/df/7207c44b8644d9a0d8eeac60b58dde9e184b905e47db2391e6ca751a8701/http_exceptions-0.2.9-py3-none-any.whl#sha256=7cffe6b9f39130f2ce13f66b4bb376ce61dde6f435cc2385eb6f674f48ac9634" data-requires-python="&gt;=3.7,&lt;4.0" >http_exceptions-0.2.9-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/1a/ee/eed139b5dcd4019d1148eebffb199a22320b9e46e268de3bffcca9a9012c/http-exceptions-0.2.10.tar.gz#sha256=77484475de13ead5dac72816387dd3c5d3898fd2588af02c1cd5454fb7412c06" data-requires-python="&gt;=3.7,&lt;4.0" >http-exceptions-0.2.10.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/9d/ca/53101f30b30fe3c6c247aced01aeda263ef4ce8c8dad44f2d0d591a4f0b3/http_exceptions-0.2.10-py3-none-any.whl#sha256=26c97d021f798f8e6c9db477385d2e8f49ae2b26e6acb92b857637fb21cb1e2e" data-requires-python="&gt;=3.7,&lt;4.0" >http_exceptions-0.2.10-py3-none-any.whl</a><br />
</body>
</html>
<!--SERIAL 12897601-->